### ISSUE TITLE
Can't find variable error since Angular v15 upgrade on iOS 15 devices

### ISSUE THREAD

**Issue by VictorienTardif** [angular/angular#48300]
### Which @angular/* package(s) are the source of the bug?

Don't known / other

### Is this a regression?

No

### Description

JS is blocked by a `ReferenceError: Can't find variable` error since Angular v15 upgrade on iOS 15 devices, on any browser.

The reproduction link is easy to provide, you have the bug on the [angular.io](https://angular.io/) website.
Open it on a iOS 15 device to discover a full white screen. See https://github.com/angular/angular/issues/48296

Here is a screenshot of the console on the Angular website:
![f4f6024e-687e-441d-ad14-3b0fe821a573](https://user-images.githubusercontent.com/3167791/204869850-131f3986-ba06-48ea-add5-018ff7e74c41.jpg)

### Please provide a link to a minimal reproduction of the bug

https://angular.io/

**Comment by davebray131** [comment]
I am also experiencing this issue in all my projects.  I am going to revert them back to Angular 14 for now, but if I can be of assistance in troubleshooting please let me know.

**Comment by alan-agius4** [comment]
Likely a duplicate of https://github.com/angular/angular/issues/48130
